src/model_header.ODE_template.h

```diff
@@ -733,6 +733,10 @@ class Model_TPL_MODELNAME : public amici::Model_ODE {
     TPL_X_SOLVER_IMPL
 
     TPL_TOTAL_CL_IMPL
+    
+    virtual std::string getName() const override {
```

Comment:
```suggestion
    std::string getName() const override {
```

Redundant for overrides.